google/cloud/bigtable/internal/instance_admin_test.cc

```diff
@@ -591,8 +649,10 @@ auto create_list_app_profiles_lambda =
         EXPECT_NE(nullptr, response);
         for (auto const& app_profile_id : app_profile_ids) {
           auto& app_profile = *response->add_app_profiles();
-          app_profile.set_name(instance_name + "/appProfiles/" +
-                               app_profile_id);
+          auto name = instance_name;
```

Comment:
Combine into one line? Or are the `+=` appends more efficient?

Reply:
According to clang-tidy (on my IDE, I do not believe we have turned that on for the CI builds) they are.  I do not know this to be true for a fact, but keeping clang-tidy happy seems worthwhile.

Reply:
Interesting; I see the concatenations with `+` on a single line elsewhere, but multiple lines with `+=` here; would be curious some day to find out why they're treated differently. Not suggesting any action items for you; just thinking out loud.